### DISCUSSION TITLE
能否将设置独立成库

### DISCUSSION THREAD

**Discussion by ivysrono** [the1812/Bilibili-Evolved#3063] [功能建议 / Ideas]
现在大多用 https://github.com/sizzlemctwizzle/GM_config/ 但年久失修实在太丑。
作者的设置界面是我看过的功能最丰富外表最美观的GM脚本设置，不知能否独立成库以供调用？

**Comment by the1812** [comment]
短期内不会, 功能还很简陋, 接口最近也还在改造
